changes/2020-07-06_clarify-streaming-encrypt-decrypt/change.md

```diff
@@ -166,8 +214,6 @@ as that would defeat the point of streaming.
 Thus, the plaintext needs to go somewhere other than memory.
 Because this is a client-side library, we MUST give it to the customer.
 
-In the case of any decryption failure,
-the operation MUST zero out any available but unreleaed plaintext.
```

Comment:
Just a note that I cut this out now that I've slightly tweaked the definition for release (also realized that this was in the change doc but not the spec). I'm opting to not prescribe what to do for failure cases since what you can or should do will depend on the streaming framework.